pkg/sentry/server/server.go

```diff
@@ -101,6 +101,11 @@ func (s *server) getServerCertificate() (*tls.Certificate, error) {
 	now := time.Now().UTC()
 	issuerExp := s.certAuth.GetCACertBundle().GetIssuerCertExpiry()
 	serverCertTTL := issuerExp.Sub(now)
+	hoursLeft := serverCertTTL.Hours()
+
+	if serverCertTTL.Hours() <= 720 {
+		log.Warn("The root CA cert is about to expire in %s days", hoursLeft/24)
+	}
```

Comment:
Some minor changes. You should use `Warnf` here, and `%f` to format this float value.
```
        if hoursLeft := serverCertTTL.Hours(); hoursLeft <= 720 {
		log.Warnf("The root CA cert is about to expire in %f days", hoursLeft/24)
	}
```